src/components/cardbuilder/cardImage.ts

```diff
@@ -0,0 +1,61 @@
+import type { BaseItemDto } from '@jellyfin/sdk/lib/generated-client/models/base-item-dto';
+import type { ApiClient } from 'jellyfin-apiclient';
+import type { CardOptions } from 'types/cardOptions';
+import { CardShape } from 'utils/card';
+
+import { getCardImageUrl } from './cardBuilder';
+
+/**
+ * Builds an html string for a basic image only card.
+ */
+export function buildCardImage(
+    apiClient: ApiClient,
+    item: BaseItemDto,
+    options: CardOptions
+): string {
+    let shape: CardShape = CardShape.Square;
+    if (item.PrimaryImageAspectRatio) {
+        if (item.PrimaryImageAspectRatio >= 3) {
+            shape = CardShape.Banner;
+        } else if (item.PrimaryImageAspectRatio >= 1.33) {
+            shape = CardShape.Backdrop;
+        } else if (item.PrimaryImageAspectRatio > 0.71) {
+            shape = CardShape.Square;
+        } else {
+            shape = CardShape.Portrait;
+        }
+    }
+
+    const image = getCardImageUrl(
+        item,
+        apiClient,
+        options,
+        shape
+    );
+
+    if (!image) return '';
+
+    const className = ` ${shape}Card`;
+
+    const { blurhash, imgUrl } = image;
+    let blurhashAttrib = '';
+    if (blurhash && blurhash.length > 0) {
+        blurhashAttrib = `data-blurhash="${blurhash}"`;
+    }
+
+    return (
+        `<div class="card ${className}">
+    <div class="cardBox">
+        <div class="cardScalable">
+            <div class="cardPadder cardPadder-${shape}"></div>
```

Comment:
Should we add an [icon](https://github.com/jellyfin/jellyfin-web/blob/408b7c011a8534b58d99e0a5ffead5ef9b506ec3/src/components/cardbuilder/cardBuilder.js#L1112-L1119) when blurhash is disabled?

Reply:
I (finally) updated this to include the icon logic.